pkg/processor/runtime/context.go

```diff
@@ -25,11 +25,7 @@ import (
 )
 
 func newContext(parentLogger nuclio.Logger, configuration *Configuration) (*nuclio.Context, error) {
-	newContext := &nuclio.Context{
-		Logger:      parentLogger,
-		DataBinding: map[string]nuclio.DataBinding{},
-	}
-
+	dataBindings := map[string]nuclio.DataBinding{}
```

Comment:
1. What exactly is the behavioral change here?
2. Comments should always have a newline before them

Reply:
I was wrong when I said nuclio-sdk didn't change - it did (dataBinding became a map). I just pushed to nuclio-sdk.